File: Java/src/pagina66b.java
//Programa que apresenta os resultados de uma tabuada de multiplicar (de 1 até 10) de um número qualquer.

import java.util.Scanner;

public class pagina66b {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite um número: ");
        int tabuada = scanner.nextInt();

        for(int i = 0; i <= 10; i++){
            System.out.println(tabuada+" X "+i+" = "+i*tabuada);
        }

        scanner.close();
    }
}

File: JavaScript/Manzano/Pagina66b.js
// Programa que apresenta os resultados de uma tabuada de multiplicar (de 1 até 10) de um número qualquer.

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Digite um número: ", function (input) {
  const tabuada = parseInt(input);

  for (let i = 0; i <= 10; i++) {
    console.log(tabuada + " X " + i + " = " + (i * tabuada));
  }

  rl.close();
});